driver-core/src/main/com/mongodb/internal/binding/SingleServerBinding.java

```diff
@@ -129,4 +129,3 @@ public void release() {
         }
     }
 }
-
```

Comment:
The IDE added the line because at some point I touched the file, then reverted, but the IDE follows the rules set in `.editorconfig`. Now I tried reverting via a more dumb text editor, but the static checks fail (the rule was apparently introduced after committing this file without a line-end character).